src/ert/scheduler/lsf_driver.py

```diff
@@ -283,5 +331,56 @@ async def _process_job_update(self, job_id: str, new_state: AnyJob) -> None:
                 del self._iens2jobid[iens]
             await self.event_queue.put(event)
 
+    async def _poll_once_by_bhist(self, missing_job_ids: Iterable[str]) -> _Stat:
```

Comment:
When I look at the old `lsf_driver.cpp` there we poll immediately two times in a row, I guess here it will be tackled by the `def poll(self)` function. `self._bhist_cache_timestamp` is used to determine whether the bhist_cache is not too old, but it is set only in the constructor. Shouldn't it be set everytime we call it?

Reply:
ouch, good catch, that would be catastrophic..

Reply:
Fixed in dfc1bd2